src/contracts/pods/EigenPod.sol

```diff
@@ -467,6 +516,13 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         delayedWithdrawalRouter.createDelayedWithdrawal{value: amountWei}(podOwner, recipient);
     }
 
+    function _effectiveRestakedBalanceGwei(uint64 amountGwei) internal returns (uint64){
+        //calculates the "floor" of amountGwei - EFFECTIVE_RESTAKED_BALANCE_OFFSET
+        uint64 effectiveBalance = uint64((amountGwei - EFFECTIVE_RESTAKED_BALANCE_OFFSET) / GWEI_TO_WEI * GWEI_TO_WEI);
```

Comment:
i understand this a bit, but could you explain it further in comments

Reply:
added an explainer.